src/Hosting/Hosting/src/Internal/ConsoleLifetime.cs

```diff
@@ -19,14 +19,15 @@ public class ConsoleLifetime : IHostLifetime, IDisposable
         private CancellationTokenRegistration _applicationStartedRegistration;
         private CancellationTokenRegistration _applicationStoppingRegistration;
 
-        public ConsoleLifetime(IOptions<ConsoleLifetimeOptions> options, IHostEnvironment environment, IHostApplicationLifetime applicationLifetime)
-            : this(options, environment, applicationLifetime, NullLoggerFactory.Instance) { }
+        public ConsoleLifetime(IOptions<ConsoleLifetimeOptions> options, IHostEnvironment environment, IHostApplicationLifetime applicationLifetime, IOptions<HostOptions> hostOptions)
```

Comment:
Changing this signature requires updating ref assemblies.
https://github.com/aspnet/Extensions/blob/master/docs/ReferenceAssemblies.md

Reply:
expected to change both netstandard 2.0 and netcoreapp3.0? or just netcoreapp3.0? the script auto-populated for both.

Reply:
Yeah, both.